host/xtest/regression_4000.c

```diff
@@ -5436,6 +5436,9 @@ static void xtest_tee_test_4011(ADBG_Case_t *c)
 
 		/* 4.3 */
 		n = n + i + tmp_size - m;
+		if (n >= tmp_size)
+			goto out;
```

Comment:
if this is an unexpected case, the test should fail, i.e.
```c
		if (!ADBG_EXPECT_COMPARE_UNSIGNED(c, n, <, tmp_size))
			goto out;

Reply:
By the way, `n == tmp_size` is a valid case, no?
If so, replace `<` with `<=`.

Reply:
> By the way, `n == tmp_size` is a valid case, no?
> If so, replace `<` with `<=`.

Will leave to Rajesh to comment.

Reply:
Rajesh doesn't seem to comment. How can we move forward?

Reply:
> Rajesh doesn't seem to comment. How can we move forward?

We can always continue working with the patch on our own.